src/main/kotlin/com/jobtracker/api/security/IssuerValidator.kt

```diff
@@ -0,0 +1,19 @@
+package com.jobtracker.api.security
+
+import org.springframework.security.oauth2.core.OAuth2Error
+import org.springframework.security.oauth2.core.OAuth2TokenValidator
+import org.springframework.security.oauth2.core.OAuth2TokenValidatorResult
+import org.springframework.security.oauth2.jwt.Jwt
+
+/**
+ * Validates JWT against issuer
+ */
+class IssuerValidator(private val issuer: String) : OAuth2TokenValidator<Jwt> {
+    override fun validate(token: Jwt): OAuth2TokenValidatorResult {
```

Comment:
It looks like this won't distinguish between a 401 Unauthorized and a 403 forbidden.
_Technically,_ we should care about this distinction if we want to be RESTful compliant, but truthfully, I would be OK with if we simply stick to 403. (Pending confirmation from you and Nic)